samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/CallingBot.cs

```diff
@@ -140,6 +140,11 @@ private async Task NotificationProcessor_OnNotificationReceivedAsync(Notificatio
                         });
                         await callService.Record(callId, audioRecordingConstants.PleaseRecordYourMessage);
                     }
+
+                    if (callBotCache.TryGetValue($"{callId}:incident", out IncidentDetails incidentDetails))
```

Comment:
This cache is now used for multiple scenarios-- is it time to give it a proper interface vs using IMemoryCache directly?

Reply:
good point made changes